libr/cons/cons.c

```diff
@@ -138,7 +138,8 @@ static inline void r_cons_write(const char *buf, int len) {
 	if (I.fdout < 1) {
 		I.fdout = 1;
 	}
-	(void) write (I.fdout, buf, len);
+	int r = write (I.fdout, buf, len);
+	(void)r;
```

Comment:
wtf is this dead code

Reply:
Is to silent warnings ... (void)write ... is not enough in some cases.

Reply:
Fair enough, I was not too proud of it either...